src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParserHelperClasses.cs

```diff
@@ -1343,19 +1343,14 @@ internal SqlReturnValue() : base()
         }
     }
 
-    internal class MultiPartTableName
+    internal struct MultiPartTableName
```

Comment:
Any particular reason for this? It'll change allocation semantics though I don't know whether for better or worse.

Reply:
It was originally a struct. I changed it to a class but it required too many other changes so I left it as a struct for now.